doc/users/pyplot_tutorial.rst

```diff
@@ -73,10 +74,11 @@ several ways to set line properties
       plt.plot(x, y, linewidth=2.0)
 
 
-* Use the setter methods of the ``Line2D`` instance.  ``plot`` returns a list
-  of lines; e.g., ``line1, line2 = plot(x1,y1,x2,y2)``.  Below I have only
-  one line so it is a list of length 1.  I use tuple unpacking in the
-  ``line, = plot(x, y, 'o')`` to get the first element of the list::
+* Use the setter methods of a ``Line2D`` instance.  ``plot`` returns a list
+  of ``Line2D`` objects; e.g., ``line1, line2 = plot(x1,y1,x2,y2)``.  In the code
```

Comment:
We could be more consistent in formatting by adding spaces: `plot(x1, y1, x2, y2)`.